src/main/java/stsjorbsmod/patches/BottledMemoryPatch.java

```diff
@@ -0,0 +1,113 @@
+package stsjorbsmod.patches;
+
+import com.badlogic.gdx.graphics.g2d.SpriteBatch;
+import com.evacipated.cardcrawl.modthespire.lib.SpireField;
+import com.evacipated.cardcrawl.modthespire.lib.SpirePatch;
+import com.evacipated.cardcrawl.modthespire.lib.SpirePostfixPatch;
+import com.megacrit.cardcrawl.cards.AbstractCard;
+import com.megacrit.cardcrawl.cards.CardGroup;
+import com.megacrit.cardcrawl.characters.AbstractPlayer;
+import com.megacrit.cardcrawl.core.Settings;
+import com.megacrit.cardcrawl.helpers.RelicLibrary;
+import com.megacrit.cardcrawl.relics.AbstractRelic;
+import com.megacrit.cardcrawl.relics.BottledFlame;
+import com.megacrit.cardcrawl.relics.BottledLightning;
+import com.megacrit.cardcrawl.relics.BottledTornado;
+import javassist.CannotCompileException;
+import javassist.expr.ExprEditor;
+import javassist.expr.FieldAccess;
+import javassist.expr.MethodCall;
+import stsjorbsmod.relics.BottledMemoryRelic;
+
+public class BottledMemoryPatch {
```

Comment:
Based on searching for other base game references to `inBottleFlame`/etc, I think we're going to want a few more patches:

* [ ] `Duplicator`, `DollysMirror`: Need to special case these to reset the `inBottleMemory` field back to false after they duplicate their target card
* [ ] `FountainOfCurseRemoval`: Need to special case to avoid picking bottle cards

Reply:
Sort of pretending that the FountainOfCurseRemoval actually did the right thing. ran through 250ish floors without seeing it, gave up.